README.txt

NAME

Bundle::STBEY::Favourites - a Bundle of my favourite modules

DESCRIPTION

This module only serves the purpose of automatically
installing the following modules as "prerequisites"
when this module is installed with "CPAN" or "CPANPLUS":

CONTENTS

Carp::Clan        5.3
Bit::Vector       7.1
Date::Calc        6.0
Date::Calc::Util  1.0
Date::Pcalc       6.0
Data::Locations   5.5
Math::MatrixBool  5.8
Set::IntRange     5.2
Bundle::libwin32
Bundle::CPAN
CPAN::Reporter
Bundle::libnet
Bundle::LWP
Parse::RecDescent
Digest::MD5
Digest::SHA1
Unicode::String
IO::Stringy
MIME::Parser
Getopt::Long
MIME::Base64
MIME::Tools
Time::HiRes
Storable
HTML::Parser
MIME::Lite

VERSION

This man page documents "Bundle::STBEY::Favourites" version 1.1.

AUTHOR

  Steffen Beyer
  mailto:STBEY@cpan.org
  http://www.engelschall.com/u/sb/download/

COPYRIGHT

Copyright (c) 2009 by Steffen Beyer. All rights reserved.

LICENSE

This package is free software; you can redistribute it
and/or modify it under the same terms as Perl itself.

DISCLAIMER

This package is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.



CHANGES.txt
             ===================================================
               Package "Bundle::STBEY::Favourites" Version 1.1
             ===================================================


                     Copyright (c) 2009 by Steffen Beyer.
                             All rights reserved.


Version history:
----------------

Version 1.1   08.10.2009

 +  Added Bundle::libwin32

Version 1.0   07.10.2009

 +  Initial release

----------------------------------------------------------------------------



lib/Bundle/STBEY/Favourites.pm

###############################################################################
##                                                                           ##
##    Copyright (c) 2009 by Steffen Beyer.                                   ##
##    All rights reserved.                                                   ##
##                                                                           ##
##    This package is free software; you can redistribute it                 ##
##    and/or modify it under the same terms as Perl itself.                  ##
##                                                                           ##
###############################################################################

package Bundle::STBEY::Favourites;

use strict;

use vars qw(@ISA @EXPORT @EXPORT_OK %EXPORT_TAGS $VERSION);

require Exporter;
require DynaLoader;

@ISA = qw(Exporter DynaLoader);

@EXPORT = qw();

@EXPORT_OK = qw();

%EXPORT_TAGS = (all => [@EXPORT_OK]);

$VERSION = '1.1';

sub Version { return $VERSION; }

1;

__END__

=pod

=head1 NAME

Bundle::STBEY::Favourites - a Bundle of my favourite modules

=head1 DESCRIPTION

This module only serves the purpose of automatically
installing the following modules as "prerequisites"
when this module is installed with "CPAN" or "CPANPLUS":

=head1 CONTENTS

Carp::Clan        5.3
Bit::Vector       7.1
Date::Calc        6.0
Date::Calc::Util  1.0
Date::Pcalc       6.0
Data::Locations   5.5
Math::MatrixBool  5.8
Set::IntRange     5.2
Bundle::libwin32
Bundle::CPAN
CPAN::Reporter
Bundle::libnet
Bundle::LWP
Parse::RecDescent
Digest::MD5
Digest::SHA1
Unicode::String
IO::Stringy
MIME::Parser
Getopt::Long
MIME::Base64
MIME::Tools
Time::HiRes
Storable
HTML::Parser
MIME::Lite

=head1 VERSION

This man page documents "Bundle::STBEY::Favourites" version 1.1.

=head1 AUTHOR

  Steffen Beyer
  mailto:STBEY@cpan.org
  http://www.engelschall.com/u/sb/download/

=head1 COPYRIGHT

Copyright (c) 2009 by Steffen Beyer. All rights reserved.

=head1 LICENSE

This package is free software; you can redistribute it
and/or modify it under the same terms as Perl itself.

=head1 DISCLAIMER

This package is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.



Makefile.PL
#!perl -w

###############################################################################
##                                                                           ##
##    Copyright (c) 2009 by Steffen Beyer.                                   ##
##    All rights reserved.                                                   ##
##                                                                           ##
##    This package is free software; you can redistribute it                 ##
##    and/or modify it under the same terms as Perl itself.                  ##
##                                                                           ##
###############################################################################

use strict;

use ExtUtils::MakeMaker;

use Config;

WriteMakefile(
    'NAME'              => 'Bundle::STBEY::Favourites',
    'VERSION_FROM'      => 'lib/Bundle/STBEY/Favourites.pm',
    'PREREQ_PM'         =>
                              {
                                  'Carp::Clan'        => 5.3,
                                  'Bit::Vector'       => 7.1,
                                  'Date::Calc'        => 6.0,
                                  'Date::Calc::Util'  => 1.0,
                                  'Date::Pcalc'       => 6.0,
                                  'Data::Locations'   => 5.5,
                                  'Math::MatrixBool'  => 5.8,
                                  'Set::IntRange'     => 5.2,
                                  'Bundle::libwin32'  => 0,
                                  'Bundle::CPAN'      => 0,
                                  'CPAN::Reporter'    => 0,
                                  'Bundle::libnet'    => 0,
                                  'Bundle::LWP'       => 0,
                                  'Parse::RecDescent' => 0,
                                  'Digest::MD5'       => 0,
                                  'Digest::SHA1'      => 0,
                                  'Unicode::String'   => 0,
                                  'IO::Stringy'       => 0,
                                  'MIME::Parser'      => 0,
                                  'Getopt::Long'      => 0,
                                  'MIME::Base64'      => 0,
                                  'MIME::Tools'       => 0,
                                  'Time::HiRes'       => 0,
                                  'Storable'          => 0,
                                  'HTML::Parser'      => 0,
                                  'MIME::Lite'        => 0

                              },
#   ($] >= 5.005 ?
#       ('ABSTRACT'     => 'Bundle of my favourite modules',
#        'AUTHOR'       => 'Steffen Beyer <STBEY@cpan.org>') : ()),
#   ($] >= 5.005 && $^O eq 'MSWin32' && $Config{archname} =~ /-object\b/i ?
#       ('CAPI'         => 'TRUE') : ()),
    'dist'              => { COMPRESS => "gzip -9", SUFFIX => "gz" }
);

__END__

